homeassistant/components/netatmo/camera.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         self._camera = cast(NaModules.Camera, netatmo_device.device)
         self._id = self._camera.entity_id
         self._home_id = self._camera.home.entity_id
+        self._attr_name = self._camera.name
         self._device_name = self._camera.name
```

Comment:
```suggestion
        self._device_name = self._attr_name = self._camera.name
```

And please remove line 86 `_attr_has_entity_name = True`